NameLayerMain/src/vg/civcraft/mc/namelayer/GroupManager.java

```diff
@@ -169,27 +198,19 @@ public void mergeGroup(final Group group, final Group toMerge){
 		if (toMerge.getSuperGroup() != null) {
 			Group sup = toMerge.getSuperGroup();
 			Group.unlink(sup, toMerge); // need to unlink any supergroup from merge.
-			invalidateCache(sup.getName());
```

Comment:
Little concerned at this removal. 

I added these cache invalidations because without them the group data cached doesn't update properly or completely with the merge / group changes. Do you have something else that performs the same behavior, or have you tightened something up where you are confident nothing is missed in cache transform?